pinecone/sparse/splade/splade_quora.ipynb

```diff
@@ -0,0 +1,2438 @@
+{
```

Comment:
<p>Now let's upsert vectors to the index. We are using async upload with batching. For more imformation on performance boosting, visit the Pinecone documentation on <a href="https://docs.pinecone.io/docs/performance-tuning" target="_blank">performance tuning</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>